src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -83,7 +83,8 @@
   <Target Name="CreateTestListJson"
           DependsOnTargets="CreateFuncTestListJson;CreatePerfTestListJson" />
 
-  <Target Name="CreateFuncTestListJson" DependsOnTargets="CreateAzureStorage">
+  <!-- allow skipping this target if perf testing is enabled -->
+  <Target Name="CreateFuncTestListJson" DependsOnTargets="CreateAzureStorage" Condition="'$(Performance)' != 'true' or '$(FuncTestsDisabled)' != 'true'">
```

Comment:
I will move the properties to new lines to shorten the overall length.

Regarding negative property, as the behavior is to be opt-out it feels overly verbose to conditionally set it to true, plus it is also the pattern used in other parts of build tools (e.g. disabling tests post-build follows the same pattern).